examples/with-typescript/typings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react": "registry:npm/react#15.0.1+20170104200836"
```

Comment:
i would suggest using `@types/react` directly in NPM instead of relying on `typings`. 
it's the way of the future, and you wouldn't need `postinstall` task.